core/options.go

```diff
@@ -57,6 +57,7 @@ type ServiceOptions struct {
 
 	// Host string resolved to an IP, including DNS lookup.
 	host net.IP
+	DelIfAddr bool
```

Comment:
Since this field is not participating in REST API, therefore should be invisible to JSON marshaller, can you make it lowercase?